src/CoreGraphics/CGColorConversionInfo.cs

```diff
@@ -145,14 +145,14 @@ public CGColorConversionInfo (CGColorSpace src, CGColorSpace dst)
 		[iOS (13,0)]
 		[TV (13,0)]
 		[Watch (6,0)]
-		public CGColorConversionInfo (CGColorSpace src, CGColorSpace dst, NSDictionary options)
+		public CGColorConversionInfo (CGColorSpace source, CGColorSpace destination, NSDictionary options)
 		{
-			if (src == null)
-				throw new ArgumentNullException (nameof (src));
-			if (dst == null)
-				throw new ArgumentNullException (nameof (dst));
+			if (source == null)
+				throw new ArgumentNullException (nameof (source));
+			if (destination == null)
+				throw new ArgumentNullException (nameof (destination));
 
-			Handle = CGColorConversionInfoCreateWithOptions (src.Handle, dst.Handle, options.GetHandle ());
+			Handle = CGColorConversionInfoCreateWithOptions (source.Handle, destination.Handle, options.GetHandle ());
```

Comment:
I think there is a potential NRE here

```
Handle = CGColorConversionInfoCreateWithOptions (source.Handle, destination.Handle, options.GetHandle ());
```

`options` can be `null`

Reply:
@dalexsoto no, `GetHandle` is an extension method to avoid that :)

Reply:
Ahh I forgot about it :) thanks!